github/commands/pull_request.py

```diff
@@ -131,3 +131,25 @@ def view_diff_for_pr(self):
 
     def open_pr_in_browser(self):
         open_in_browser(self.pr["html_url"])
+
+
+class GsCreatePullRequestCommand(TextCommand, GitCommand, git_mixins.GithubRemotesMixin):
+    """
+    Create pull request of the current commit on the current repo.
+    """
+
+    def run(self, edit):
+        sublime.set_timeout_async(self.run_async, 0)
+
+    def run_async(self):
+        base_remote = github.parse_remote(self.get_integrated_remote_url())
+        remote_branch = self.get_active_remote_branch()
```

Comment:
There should probably be a check here to abort if there was an issue identifying the remote branch. i.e.
```python
if not remote_branch:
    sublime.message_dialog("Unable to determine remote.")
```
We can swap this out for more intelligent behavior later.